tensorboard/webapp/metrics/store/metrics_reducers.ts

```diff
@@ -852,7 +859,40 @@ const reducer = createReducer(
         }
       }
 
-      const nextState = {
+      if (response.runToSeries) {
```

Comment:
Thinking a little more about this, especially in light of the changes in https://github.com/tensorflow/tensorboard/pull/6175.

The isStepSelectorEnabled state is no longer a global indicator of whether we will show step selection for *all* cards or *zero* cards. You can have the checkbox turned "off" but still show step selection on a specific card. Likewise, you can have the checkbox turned "on" but still not show step selection on a specific card. This is something we have to keep in mind for this new ngrx-based time selection state, too, and I don't think we are taking that into account with these changes.

The way things are written now I suspect you planned to have timeSelection for all cards and just determine whether to show the steps in a card based on the isStepSelectorEnabled state.